pkg/upgraders/healthcheckstep.go

```diff
@@ -40,6 +41,18 @@ func (c *clusterUpgrader) PreUpgradeHealthCheck(ctx context.Context, logger logr
 		}
 		return false, err
 	}
+
+	if c.upgradeConfig.Spec.CapacityReservation {
+		ok, err := c.scaler.CanScale(c.client, logger)
+		if !ok {
+			c.metrics.UpdateMetricHealthcheckFailed(c.upgradeConfig.Name, metrics.DefaultWorkerMachinepoolNotFound)
```

Comment:
So currently, there's no notification (`c.notifier.Notify(notifier.MuoStateHealthCheck)`) sent for this failure in pre-healthcheck. Is that missed?